Public/Src/FrontEnd/Script/Ambients/Transformers/AmbientTransformer.Process.cs

```diff
@@ -507,6 +507,9 @@ private void ProcessExecuteArguments(Context context, ObjectLiteral obj, Process
             }
             processBuilder.AdditionalTempDirectories = ProcessOptionalPathArray(obj, m_executeAdditionalTempDirectories, strict: false, skipUndefined: true);
 
+
+            // Set the default value before processing unsafeOption in case unsafeOption doesn't exist.
+            processBuilder.Options |= Process.Options.RequireGlobalDependencies;

```

Comment:
>processBuilder.Options |= Process.Options.RequireGlobalDependencies; [](start = 12, length = 68)

I'm not sure what you're trying to achieve here. Why do we enable it by default here, and then possibly disable it later?

Reply:
We are trying to make the feature opt-in. So the logic is enable it first. Later on if it is specified in the unsafe option to opt it out, we disable it.